deployment/helm-k8s/Chart.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 description: Siembol is an open-source, real-time Security Information & Event Management tool based on big data technologies, providing a scalable, advanced security analytics framework
 name: siembol
 type: application
-version: 0.1.4
+version: 0.1.5
```

Comment:
Do we need increase the version? have we release 0.1.4?

Reply:
you're right.